frontend/src/features/Tweets/Footer/Comment.test.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { render, screen, fireEvent } from "@testing-library/react";
+import Comment from "./Comment";
+
+const mockOnSubmit = jest.fn();
+const mockSetForm = jest.fn();
+
+const initialForm = {
+  isOpen: false,
+  comment: "",
+};
+
+describe("Comment", () => {
+  beforeEach(() => {
+    mockOnSubmit.mockClear();
+    mockSetForm.mockClear();
+
+    render(
+      <Comment
+        commentsAmount={5}
+        onSubmit={mockOnSubmit}
+        form={initialForm}
+        setForm={mockSetForm}
+      />
+    );
+  });
+
+  it("Show how many comment have", () => {
+    expect(screen.getByText("5")).toBeInTheDocument();
```

Comment:
The test `screen.getByText("5")` is too broad because it could match any occurrence of the number "5" on the screen, leading to false positives. We need to ensure we're targeting the correct element representing the comment count.

To improve this:

* Use a more specific selector, such as a `data-testid` or a role/label that uniquely identifies the comment count.

* For example, if the comment count is inside a specific element, you could do something like:

```js
expect(screen.getByTestId("comment-count")).toHaveTextContent("5");
```

This makes the test more reliable and ensures it only targets the comment count, reducing the risk of errors.